app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsSecurityScreen.kt

```diff
@@ -28,14 +28,26 @@ object SettingsSecurityScreen : SearchableSettings {
 
     @Composable
     override fun getPreferences(): List<Preference> {
-        val context = LocalContext.current
         val securityPreferences = remember { Injekt.get<SecurityPreferences>() }
-        val authSupported = remember { context.isAuthenticationSupported() }
+        return listOf(
+            getSecurityGroup(securityPreferences),
+            getFirebaseGroup(securityPreferences),
+        )
+    }
+}
 
-        val useAuthPref = securityPreferences.useAuthenticator()
-        val useAuth by useAuthPref.collectAsState()
+@Composable
+private fun getSecurityGroup(
+    securityPreferences: SecurityPreferences,
+): Preference.PreferenceGroup {
```

Comment:
Make this a function of `SettingsSecurityScreen`

Reply:
I'm not sure I fully understand what you mean.

Reply:
Function of the object. Currently it's a top level function